KMaina/Fast-Food-App
# #160739612 Redesign the Order Approve page

#### What does this PR do?
Allows an admin to approve or decline and orders
#### Description of Task to be completed?
The admin should either check the accepted or decline radio button
#### How should this be manually tested?

- Clone the repo onto your local machine.
- Open a terminal and run cd Fast-Food-App
- run `git clone https://github.com/KMaina/Fast-Food-App.git`
- run `git checkout ch-admin-order-page-160739612 `
- In the folder where you cloned the repo, you should open the UI folder.
- Open the admin-order-edit.html to see it in your browser for review.

#### Any background context you want to provide?
This is a redesign of the page
#### What are the relevant pivotal tracker stories?
#160739612
